docs/manual/extensions/maklermodul.de.md

```diff
@@ -26,12 +26,16 @@ Diese Themes liefern bereits Layoutvorlagen (CSS und Templates) für das Maklerm
 **Dokumentation:** [docs.contao-themes.net](https://docs.contao-themes.net/#/mate_theme/maklermodul)  
 **Kostenlos installieren oder Lizenz bestellen:** [contao-themes.net/theme-detail/mate.html](https://contao-themes.net/theme-detail/mate.html)  
 
-![Immobilienliste mit Auswahllisten im MATE Theme](/de/extensions/images/de/maklermodul_mate_liste_selects.png ':size=150') ![Immobilienliste mit Buttons im MATE Theme](/de/extensions/images/de/maklermodul_mate_liste_buttons.png ':size=150') ![Immobilien Exposé im MATE Theme](/de/extensions/images/de/maklermodul_mate_details.png ':size=150')
+![Immobilienliste mit Auswahllisten im MATE Theme](/de/extensions/images/de/maklermodul_mate_liste_selects.png ':size=150') 
```

Comment:
Can you please remove « ':size=150'» in every image tag. If it is really necessary to resize the image, you will find the correct syntax in the [documentation](https://themes.gohugo.io//theme/hugo-theme-learn/en/cont/markdown/#resizing-image).